app/Http/routes.php

```diff
@@ -36,6 +36,10 @@
 
 Route::get('/icons', 'HomeController@getIcons');
 
+// featured artists
+Route::get('/beatmaps/artists', ['as' => 'artist.index', 'uses' => 'ArtistsController@index']);
+Route::get('/beatmaps/artists/{artist_id}', ['as' => 'artist.show', 'uses' => 'ArtistsController@show']);
```

Comment:
Route::resource('beatmaps/artists', 'ArtistsController', ['only' => ['index', 'show']]);

(though why is it under `/beatmaps`?)